### ISSUE TITLE
installer does not check if the root folder is writable

### ISSUE THREAD

**Issue by f1-outsourcing** [salesagility/SuiteCRM-Core#121]
Although I have all green checks in the check popup, the installation failed on 

`vendor/symfony/filesystem/Filesystem.php on line 741PHP message: PHP Warning:  rename(/xxxx/xxxxx/tmp/.env.localNs0YAk,.env.local): Permission denied in`

So maybe add writeable check on root or put env.local somewhere else.

v8.1.2

**Comment by johnM2401** [comment]
Hey

Are you still experiencing issues with this?

I haven't come across this issue myself, when installing Suite8

If you're still having issues, could you provide some more information?

Such as:
Steps to Replicate / PHP Version 

Thanks!

**Comment by f1-outsourcing** [comment]
php version???? What has that to do with it?

reproduce
```
mkdir website
cd website 
unzip ../SuiteCRM-8.1.2.zip
cd ..
chmod a-w website
```

now do the install and keep checking the php error log. You will find all directories that are needed to be writeable and the once not being checked.

**Comment by serhiisamko091184** [comment]
Hello @f1-outsourcing, 

thanks for the information, I've just installed SuiteCRM 8.1.2 from the pre-built package, and I haven't encountered the error mentioned above during the installation process. If this is still the relevant issue on your end, please provide more details for us to investigate it in depth.      

Thanks in advance. 

Regards,
Serhii

**Comment by f1-outsourcing** [comment]
If you installed the same version as I did, and you don't encounter problems, than probably your server setup is not secure.

The application needs to verify and report about directories that need write permissions. If you don't do this you assume that every webserver in the world has the same setup.
The easiest way to test is put your whole directory tree as read only.

**Comment by pgorod** [comment]
The installation checks were all redone in 8.6.0.

I don't see the point of testing this issue on 8.1.2, we should just see if the problem is still present in 8.6.0 and take things from there.

As a more general comment, getting SuiteCRM permissions right is not just a matter of checking that the root directory is writable. I agree we should improve in this area, as many people struggle with it, but it needs a more elaborate approach.

**Comment by chris001** [comment]
Definitely, it should do a writeable check on the root base directory, plus check all the permissions, and warn you when they are different than needed for the app to work.
